index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    
```

Comment:
It is not necessary to have a link to the vite favicon.

No spare newlines please.

Reply:
done, done & done